tests/stdlib/trstgen.nim

```diff
@@ -348,8 +381,11 @@ Some chapter
       -------
 
       """
-    expect(EParseError):
-      let output9Bad = rstToHtml(input9Bad, {roSupportMarkdown}, defaultConfig())
+    let output9Bad = input9bad.toHtml(
+        expectError = "input(15, 0) Error: new section expected (section " &
```

Comment:
use `"""` + dedent (note that bootstrap isn't an issue here since it's a test file)

Reply:
there are no line breaks in this long message so I can't.